README.md

```diff
@@ -23,7 +23,12 @@
 ----
 
 ## Descrizione del progetto
-TBD
+Il progetto IO:
```

Comment:
@thisisjp 
Do you think this description is fitting the current state of IO?
Do we have an official one somewhere?

Reply:
Keep in mind this description is for those one who want to know more about IO from a tech POV